test/powersystem.jl

```diff
@@ -120,6 +120,10 @@ const PS = PowerSystem
             jacobianAD, ExaPF.residual_polar!, Vm, Va,
             ybus_re, ybus_im, pbus, qbus, pv, pq, ref, nbus, ExaPF.AutoDiff.StateJacobian())
         @test jacobianAD.J ≈ J♯
+        J = ExaPF.AutoDiff.residual_jacobian_adj!(
```

Comment:
I would maybe move this code in `test/Polar/gradient.jl`

Reply:
This is gone now.